### ISSUE TITLE
[Question] Proactive messages

### ISSUE THREAD

**Issue by fercom** [tmarkovski/bot-builder-twitter-adapter#1]
Hi, is there a way of sending proactive messages within this project to the twitter users?

**Comment by tmarkovski** [comment]
Yes, it works out of the box. If you follow the example in the documentation, sending proactive messages is independent of the channel, you just need a reference to the BotAdapter.
https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-proactive-message?view=azure-bot-service-4.0&tabs=csharp

That said, the Twitter adapter should work without modifications.

**Comment by fercom** [comment]
@tmarkovski Ok, I will try it, thank you!
